Userland/Libraries/LibM/math.cpp

```diff
@@ -317,9 +317,9 @@ static FloatT internal_gamma(FloatT x) NOEXCEPT
     // These constants were obtained through use of WolframAlpha
     constexpr long long max_integer_whose_factorial_fits = (Extractor::mantissa_bits == FloatExtractor<long double>::mantissa_bits ? 20 : (Extractor::mantissa_bits == FloatExtractor<double>::mantissa_bits ? 18 : (Extractor::mantissa_bits == FloatExtractor<float>::mantissa_bits ? 10 : 0)));
```

Comment:
Given we're no longer using this, should this be removed? Or is it not being used an accident?

Reply:
This is still being used. , see line 320.

Reply:
Ah, I missed that, sorry